src/components/GenericList.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { Box, Typography, Stack } from '@mui/material'
+import COLORS from '@/styles/colors'
+import theme from '@/styles/ThemeRegistry/theme'
+
+interface DataObject {
+  entity?: string
+  content: string
+}
+
+interface GenericListProps {
+  title: string
+  data: DataObject[]
+}
+
+const GenericList: React.FC<GenericListProps> = ({ title, data }) => {
+  return (
+    <Box
+      sx={{
+        width: '282px',
+        bgcolor: COLORS.white,
+        borderRadius: theme.spacing(2),
+        marginBottom: theme.spacing(2),
+        border: `1px solid var(--grayscale-200, ${COLORS.selected})`,
+      }}
+    >
+      <Box
+        sx={{
+          padding: '16px 20px 16px 20px',
+          borderBottom: `1px solid var(--grayscale-200, ${COLORS.selected})`,
+        }}
+      >
+        <Typography variant="labelL">{title}</Typography>
+      </Box>
+
+      <Box sx={{ padding: theme.spacing(2.5) }}>
+        {data.map((item, index) => (
+          <Box
+            key={index}
+            sx={{
+              borderBottom:
+                index !== data.length - 1
+                  ? `1px solid var(--grayscale-200, ${COLORS.selected})`
```

Comment:
@asad-codingcrafts 
`var(--grayscale-200`
won't apply unless you add it to global css file as a variable.
you can use color instead